hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/queries/HapiQueryOp.java

```diff
@@ -245,7 +246,9 @@ private Transaction fittedPayment(HapiSpec spec) throws Throwable {
             long initNodePayment = costOnlyNodePayment(spec);
             Transaction payment = finalizedTxn(spec, opDef(spec, initNodePayment), true);
             if (!loggingOff) {
```

Comment:
No need for loggingOff flag. If logging is off (as configured in log4j.properties) then log.info becomes a no-op

Reply:
I didn't touch the logic